src/util/schnorr.rs

```diff
@@ -97,4 +96,11 @@ impl TweakedPublicKey {
         &self.0
     }
 
+    /// Serialize the key as a byte-encoded pair of values. In compressed form
```

Comment:
nit: Would have liked to have a sentence about 32 byte xonly keys instead of compressed keys.